docs/development/workflow/get_devel_version.rst

```diff
@@ -222,30 +222,43 @@ of `Astropy`_, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
+    There are some circumstances in which this quick method of
     activating your copy of `Astropy`_ will NOT work and you need to go
-    straight to using a virtual python environment:
+    straight to using a virtual python environment, for example:
 
-    + You use Python 3.
     + You want to work on C or Cython code in `Astropy`_.
 
+If you have decided to use the recommended "activation" method with pip, 
```

Comment:
I think this can be substantially less verbose, and it seems to me best to just start with
```
pip install -e .
```
and then below have a note that this may not work if you don't have `jinja2` and `cython` installed; if so, use `pip install jinja2`, etc. 

Alternatively, remove the first sentence and write:
"""Before trying to install, check that you have the required dependencies: "cython" and "jinja2".  If not, install them with `pip`."""

Reply:
Yup, changed according to the second option...